pallets/asset/src/lib.rs

```diff
@@ -88,6 +88,7 @@ use codec::{Decode, Encode};
 use core::mem;
 use core::result::Result as StdResult;
 use currency::*;
+use ethereum::{EcdsaSignature, EthereumAddress};
```

Comment:
can we add the `pub` before so it can be directly accessible. Because still for the metadata it is `ethereum::EcdsaSignature` Although I am 100% not sure.

Reply:
That shouldn't be necessary I think. My general preference is to avoid reexports, they tend to make refactors harder & obscure dependencies.

Reply:
Yeah I agree, But I was more concerned related to metadata that the node may be emitting, If it emits `EcdsaSignature` then everything is good.

Reply:
![Screenshot from 2021-02-25 16-57-12](https://user-images.githubusercontent.com/7436022/109147046-92fa4700-778a-11eb-9741-dc7503383d34.png)

@satyamakgec Where exactly are you seeing `ethereum::EcdsaSignature`?